Packs/Base/ReleaseNotes/1_1_15.md

```diff
@@ -0,0 +1,6 @@
+
+<!--
+#### Scripts
+##### CommonServerPython
+- Fixed an issue with mirroring mapper scheme.
+-->
```

Comment:
cant have release notes that are all commented, need to add here below the comment one of our generic rn